db/model/collector.js

```diff
@@ -124,6 +127,25 @@ module.exports = function collector(seq, dataTypes) {
           throw new ValidationError(msg);
         }
       }, // hooks.beforeUpdate
+
+      afterUpdate(inst /* , opts */) {
+        /* if status is changed to Running, then find and assign unassigned
+         generators */
+        if (inst.changed('status')) {
+          if (inst.status === collectorStatus.Running) {
```

Comment:
It would be better for this comment to be inside the "if" to match the comment below.